repairnator/docker-images/pipeline-dockerimage/Dockerfile

```diff
@@ -31,6 +31,7 @@ ENV OUTPUT=
 ENV PUSH_URL=
 ENV SMTP_SERVER=
 ENV NOTIFY_TO=
+ENV REPAIR_TOOLS=
 
```

Comment:
Simon, this should be also added in the docker file for Bears, no?

Reply:
No, it's only needed for repairnator, not for Bears.

Reply:
Oh, ok. It's because when I saw that this arg is mandatory on the launcher, I thought this would be also used for Bears. Which, in fact, I think it will for issue #498, but then I see this later, in another PR.